lib/bloc/container/container_event.dart

```diff
@@ -27,5 +27,7 @@ class DeleteContainer extends ContainerEvent {
 }
 
 class GetContainers extends ContainerEvent {
-  const GetContainers();
+  final ContainerStatus containerStatus;
+  // ignore: sort_constructors_first
```

Comment:
You don't need to ignore this warnings. The constructor should be placed in the first line in a class.